src/main/Bank.java

```diff
@@ -0,0 +1,59 @@
+package main;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bank {
+
+	private List<BankAccount> accounts = new ArrayList<>();
+	private Validator validator = new Validator();
```

Comment:
You want to validate a command and what its trying to do BEFORE it even gets to the Bank

Reply:
The bank just defines what a Bank can do. We dont want to include validation logic!